README.rst

```diff
@@ -36,3 +36,18 @@ Usage
 ------
 Execute ``linkchecker http://www.example.com``.
 For other options see ``linkchecker --help``.
+
+Docker usage
+-------------
+
+If you do not want to install any additional libraries/dependencies you can use Docker image.
```

Comment:
I'm not a native English speaker so I could be wrong, but I think there ought to be a "the" in front of "Docker image".

Reply:
I'm not a native speaker too... sorry. 
Fixed...